sonar-plugin/sonar-javascript-plugin/src/main/java/org/sonar/plugins/javascript/nodejs/NodeCommandBuilderImpl.java

```diff
@@ -182,8 +182,7 @@ private void checkNodeCompatibility(String nodeExecutable) throws NodeCommandExc
     if (!actualNodeVersion.isGreaterThanOrEqual(minNodeVersion)) {
       throw new NodeCommandException(
         String.format(
-          "Only Node.js v%s or later is supported, got %s.",
-          minNodeVersion,
+          "Unsupported Node.JS version detected %s. Please upgrade to latest Node.JS LTS version.",
```

Comment:
Grammarly agrees you are missing a 'the'
```suggestion
          "Unsupported Node.JS version detected %s. Please upgrade to the latest Node.JS LTS version.",
```